bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5947,21 +6522,33 @@ msgid "The number of analyses requested per sample type"
 msgstr "El número de análisis solicitados por tipo de muestra"
 
 #: bika/lims/content/bikasetup.py:151
-msgid "The number of days before a password expires. 0 disables password expiry"
-msgstr "Número de días antes que caduque la contraseña. Si desea que la contraseña no caduque nunca, introduzca el valor 0."
+msgid ""
+"The number of days before a password expires. 0 disables password expiry"
+msgstr ""
+"Número de días antes que caduque la contraseña. Si desea que la contraseña "
```

Comment:
@xispa I would phrase this sentence differently. Something like: _Número de días durante los que la contraseña será válida. [...]_ or _Validez de la contraseña en días. [...]_